pkg/doc/cm/credentialapplication.go

```diff
@@ -149,12 +166,21 @@ func (ca *CredentialApplication) validate() error {
 	return nil
 }
 
-func (ca *CredentialApplication) validateFormatAgainstCredManifestFormat(credManifestFormat presexch.Format) error {
+func (ca *CredentialApplication) validateFormatAgainstCredManifestFormat(cm *CredentialManifest) error {
```

Comment:
this validation is based on some rules in a specifications?

Reply:
if yes, then please mention it in comments.

Reply:
Yes, in the credential application specs. Will add a comment about it!

Reply:
Added comment [here](https://github.com/hyperledger/aries-framework-go/pull/3309/files#diff-144ca516eb99bdf97d79243f9276e01b430a1b47671b3f024753bc3bf0d63936R74)